webapp/channels/src/components/with_tooltip/with_tooltip_new/tooltip_shortcut.test.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {defineMessage} from 'react-intl';
+
+import {renderWithContext, screen} from 'tests/react_testing_utils';
+import * as userAgentUtils from 'utils/user_agent';
+
+import TooltipShortcut from './tooltip_shortcut';
+
+jest.mock('utils/user_agent', () => ({
+    isMac: jest.fn(),
+}));
+
+describe('TooltipShortcut', () => {
+    test('should show non mac shortcut when on non mac', () => {
+        (userAgentUtils.isMac as jest.Mock).mockReturnValue(false);
```

Comment:
In order to not have so many `as` in the code, I like to define a new variable as:
```
const mockedUserAgentUtils = jest.mocked(userAgentUtils);
```

Reply:
nice suggestion didn't knew about this one